Bump electron & axios

0pcom opened the pull request:
Bump electron from 15.2.0 to 22.3.25 in /electron
Bump axios from 0.21.1 to 1.6.0 in /electron/app

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/skycoin/skycoin?pullRequest=2745) <br/>All committers have signed the CLA.

Commit:
Add token permissions to CI workflow for security

GitHub recommends defining workflow permissions to secure workflows
against supply-chain attacks. This follows the principle of least
privilege by limiting the workflow to read-only access to repository
contents.

Addresses OpenSSF Scorecard Token-Permissions issue.
Inspired by PR #2670 (originally for tests.yml which is now disabled).

Commit:
Bump electron from 15.2.0 to 22.3.25 in /electron

Bumps [electron](https://github.com/electron/electron) from 15.2.0 to 22.3.25.
- [Release notes](https://github.com/electron/electron/releases)
- [Changelog](https://github.com/electron/electron/blob/main/docs/breaking-changes.md)
- [Commits](https://github.com/electron/electron/compare/v15.2.0...v22.3.25)

---
updated-dependencies:
- dependency-name: electron
  dependency-type: direct:development
...

Commit:
Bump axios from 0.21.1 to 1.6.0 in /electron/app

Bumps [axios](https://github.com/axios/axios) from 0.21.1 to 1.6.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v0.21.1...v1.6.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-type: direct:production
...
